src/main/java/uk/gov/pay/ledger/transaction/service/TransactionService.java

```diff
@@ -100,6 +100,15 @@ private TransactionSearchResponse searchTransactionsWithoutParent(TransactionSea
 
         Long total = transactionDao.getTotalForSearch(searchParams);
 
+        long size = searchParams.getDisplaySize();
+        if (total > 0 && searchParams.getDisplaySize() > 0) {
+            long lastPage = (total + size - 1) / size;
```

Comment:
This formula is less intuitive [for me].
Why not `ceil(total/size)`?

I've found different approaches here: https://stackoverflow.com/questions/7139382/java-rounding-up-to-an-int-using-math-ceil.

Reply:
We already have this in connector, didn't want to introduce something different
https://github.com/alphagov/pay-connector/blob/a6dde93f3c38f3272947fb606b053013dc223ae1/src/main/java/uk/gov/pay/connector/common/service/search/AbstractSearchStrategy.java#L21